src/wallet/wallet.cpp

```diff
@@ -2558,6 +2558,21 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
         }
     }
 
+    // Lastly, ensure it doesn't have too many ancestors
```

Comment:
nit: perhaps this comment should be "Lastly, ensure this tx will pass the mempool's chain limits"